session.go

```diff
@@ -1427,7 +1428,7 @@ func (s *session) sendPacket() (bool, error) {
 
 	if !s.handshakeConfirmed {
 		now := time.Now()
-		packet, err := s.packer.PackCoalescedPacket(protocol.MaxByteCount)
+		packet, err := s.packer.PackCoalescedPacket(s.sentPacketHandler.AmplificationWindow())
```

Comment:
Is there a risk for busy-looping here, if we exit the send loop when it's time to send an ACK? Or is this the same as if we're congestion limited, and we'll deal with that gracefully?

Reply:
This should be handled the same way as being congestion limited.